webpack.config.release.js

```diff
@@ -0,0 +1,35 @@
+const CopyPlugin = require( 'copy-webpack-plugin' );
```

Comment:
I'm using this in place of the gulp copy script to copy all the files over during the `release` step.

I don't really like it or think it's a proper use of webpack so suggestions on how to do this without this would be much appreciated.